pzsd_bot/settings.py

```diff
@@ -184,5 +198,24 @@ def get_d20(self, face: int) -> str:
 
 Emoji = _Emoji()
 
+
+class _AOCSettings(EnvSettings):
+    base_url: str = "https://adventofcode.com"
+    lb_id: str = "269893"
+    lb_path: str = f"leaderboard/private/view/{lb_id}"
+    view_key: str
+
+    rate_limit: int = 15  # in minutes
+
+    days_in_event: int = 12
+    event_start_month: int = 12
+    event_start_day: int = 1
+
+    aoc_event_active: bool = False
```

Comment:
I suppose I should add these to deployment.yaml too so they can be changed on the fly